app/models/construction.rb

```diff
@@ -13,6 +16,17 @@ class Construction < ActiveRecord::Base
   validates :start_time, presence: true
   validate :end_time_after_start_time
 
+  scope :search, -> (station_name) {
+    joins(:station_constructions)
+    .joins(:stations)
+    .merge(Station.by_name(station_name))
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 0) spaces for indenting an expression spanning multiple lines.